modules/gdscript/gdscript_parser.h

```diff
@@ -1431,7 +1431,7 @@ class GDScriptParser {
 		void push_line(const String &p_line = String());
 		void push_text(const String &p_text);
 
-		void print_annotation(AnnotationNode *p_annotation);
+		void print_annotation(const AnnotationNode *p_annotation);
```

Comment:
I made this change to allow compiling the other changes iterating with `const AnnotationNode *E`, and since the method doesn't seem to modify the node.

But I see now it makes it inconsistent with other `print_*` methods, I could change it back if needed @vnen.